webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -31,12 +31,15 @@ export type BasicContainerProps = Pick<
 >
 
 export const SectionDescription = {
+  // "Trends"
   [Section.CHECKPOINT_PLOTS]:
-    'Linear plots based on data from the experiments table.',
+    'Real-time graphs based on metrics from the Experiments Table',
```

Comment:
Let's create a ticket please to being able to include links into these descriptions.

+ keep all three sections visible, if section is empty, show the same message instead of it's content

Reply:
Do you also think "graphs" is not best here? I like it better here since trends charts are not regular DVC plots but rather an extension-specific feature I think. Or does DVCLive also update regular plots in your browser? Anyway, feel free to also commit this if you prefer so:

```suggestion
    'Real-time plots based on metrics from the Experiments Table',
```

Reply:
yes. Also not sure about making it the "Experiments Table" tbh.

Reply:
It was already "Experiments Table". That's the official name we give to that custom VS Code Editor I think?

Reply:
sorry, I meant capitalization - should it be "Experiments Table" or just "experiments table" in these descriptions?

Reply:
I like capitalization and bolding since each one has a walkthrough page. No strong opinion

BTW this page should be **Plots Dashboard**, I think (bit stronger opinion there).

Reply:
okay, let's keep it